code/game/machinery/recycler.dm

```diff
@@ -15,14 +15,17 @@ var/const/SAFETY_COOLDOWN = 100
 	var/eat_dir = WEST
 	var/amount_produced = 1
 	var/datum/material_container/materials
+	var/circuit = /obj/item/weapon/circuitboard/recycler
+	var/transmute = FALSE

```

Comment:
I know. The problem was that I needed to prevent bits of the recycler code running, while still being able to call the machinery parent.